src/modules/service-catalog/hooks/useItemFormFields.tsx

```diff
@@ -5,6 +5,8 @@ import type { TicketFieldObject } from "../../ticket-fields/data-types/TicketFie
 import type { EndUserCondition } from "../../ticket-fields/data-types/EndUserCondition";
 import { getCustomObjectKey } from "../../ticket-fields/fields/LookupField";
 import { getVisibleFields } from "../../ticket-fields/getVisibleFields";
+import DOMPurify from "dompurify";
+import { ALLOWED_HTML_TAGS } from "../../shared";
```

Comment:
We shouldn't to re-use this whole list.
My understanding is that we only need to sanitize the links as so only the `a` tags. Everything else does not seem to be a problem.
That would remove the need to move `ALLOWED_HTML_TAGS` to `shared` and simplify the approach.